integration/kubernetes/k8s-liveness-probes.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	pod_name="liveness-exec"
+}
+
+@test "Liveness probe" {
+	wait_time=120
+	sleep_time=5
+	sleep_liveness=7
+
+	# Create pod
+	sudo -E kubectl create -f pod-liveness.yaml
+
+	# Check pod creation
+	pod_status_cmd="sudo -E kubectl get pods -a | grep $pod_name | grep Running"
```

Comment:
I'm guessing maybe you can ask kubectl to just give you the status of the named pod. Not a blocker, just a thought. It can give you json output, and you can filter on what it returns I think maybe with '--selector'.